hospital/apphospital/models.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import datetime
+from django.db import models
+
+
+class Hospital(models.Model):
+    hospital_id = models.AutoField(primary_key=True)
+    hospital_name = models.CharField(max_length=200)
+    hospital_city = models.CharField(max_length=200, null=True)
+
+    def __str__(self):
+        return "ID: {} - {}".format(self.hospital_id, self.hospital_name)
+
+
+class Department(models.Model):
+    hospital = models.ForeignKey(Hospital, on_delete=models.CASCADE)
+    department_id = models.AutoField(primary_key=True)
+    department_name = models.CharField(max_length=200)
+
+    def __str__(self):
+        return "ID: {} - department {} in {}".format(self.department_id, self.department_name, self.hospital)
+
+
+class People(models.Model):
+    GENDER_TYPES = [('M', 'Male'),
+                    ('F', 'Female'),
+                    ('Other', 'Other')]
+    people_id = models.AutoField(primary_key=True)
+    people_name = models.CharField(max_length=200)
+    age = models.CharField(max_length=200)
+    gender = models.CharField(max_length=200, choices=GENDER_TYPES, default="Male")
+
+    def __str__(self):
+        return self.people_name
+
+
+class Worker(People):
+    JOBS = [('Doc', 'Doctor'),
+            ('Nur', 'Nurse')]
+    role = models.CharField(max_length=50, choices=JOBS, default='Doctor')
+    department = models.ManyToManyField(Department)
```

Comment:
`Worker` can be part of different departments but with another role.
how do you handle this senerio?

Reply:
d